gulpfile.js

```diff
@@ -0,0 +1,101 @@
+/* jshint node:true */
```

Comment:
@spencer-brown - I based off from `users` module for this file. I removed some configs like `handlebars` and `common` that is not needed here, could you take a quick look?